### DISCUSSION TITLE
When converting markdown to docx, the default table line is dashed.

### DISCUSSION THREAD

**Discussion by CBAnother** [jgm/pandoc#9538] [Q&A] [answered]
I want to turn the dashed lines in the table into solid lines.

The markdown content is as follows

| Name | Age  |
| ---- | ---- |
| Jack | 10   |
| Mary | 9    |

The commands executed are as follows.

```
pandoc "test.md" -o "test.docx"
```

The result is as shown

![image](https://github.com/jgm/pandoc/assets/62218147/e23c38fe-680a-4e43-ac70-fd0d8193c8a8)

I tried using another docx as a template, created a table within it, and set the table line through --reference-doc= while running pandoc, but it didn't work.

I want to use lua filter to change the format, but I didn't find any relevant content.

**Comment by jgm** [comment]
You need to customize the Table style in your reference.docx.

**Reply by CBAnother** [reply]
I've tried creating a new table in the reference.docx and setting it to solid lines, but the converted markdown still comes out as dashed lines.

**Reply by jgm** [reply, marked as answer]
That's not what you need to do. You need to change the Table style in the reference.docx.

**Reply by CBAnother** [reply]
Oh, I see. Add a table style named `Table`.
